packages/core/README.md

```diff
@@ -32,6 +32,12 @@ The following is a list of links to potential starting points:
 
 ## Changelog
 
+### 0.6.10 - not officially released yet
+
+* Made `DefaultPrimitiveTypeDeserializer` and `DefaultPrimitiveTypeSerializer` be able to deal with duplicate definitions of datatypes.
+  * Expose a function `shouldBeIdentical` that determines whether two datatypes should be structurally equal based on equality of: meta type, key, and language's key.
```

Comment:
If I read "structurally equal" I expect that two datatypes with the same fields would be considered equal, while from the rest of the description I think we look at the metapointers and not the structure of the datatype

Reply:
(see below)